src/rt/array/constructor.d

```diff
@@ -0,0 +1,165 @@
+/**
+ This module contains compiler support for constructing dynamic arrays
+
+  Copyright: Copyright Digital Mars 2000 - 2019.
+  License: Distributed under the
+       $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+     (See accompanying file LICENSE)
+  Source: $(DRUNTIMESRC rt/_array/_constructor.d)
```

Comment:
To maintain the "tense" (not sure if that's the right word) of the other modules in this folder, I recommend naming it "construction".

Reply:
:+1: